docs/api.md

```diff
@@ -7,21 +7,21 @@ category: 658135e7f596b800715c1cee
 ![memgpt llama](https://raw.githubusercontent.com/cpacker/MemGPT/main/docs/assets/memgpt_server.webp)
 
 > ⚠️ API under active development
-> 
+>
 > The MemGPT API is under **active development** and **breaking changes are being made frequently**. Do not expect any endpoints or API schema to persist until an official `v1.0` of the MemGPT API is released.
-> 
+>
 > For support and to track ongoing developments, please visit [the MemGPT Discord server](https://discord.gg/9GEQrxmVyE) where you can chat with the MemGPT team and other developers about the API.
-
+>
```

Comment:
I think this should have a newline between the `>` blocks? Otherwise it probably won't render properly as two callout/admonition blocks in readme (though I haven't tested it)

Reply:
You are correct, it just seems to break this convention (https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/md028.md).

Personally, I think it makes sense to be one block since these blocks have headers in them which make it clear they are two separate things.

Although, if there's strong feelings about the styling - then it should be a big deal, if anyone uses a markdown linter they can add an exception for this rule.

Reply:
I am adding a .markdownlint.json file to make an exception for this rule.

Seems to be the best solution.